src/SGB_Command_Palettes.md

```diff
@@ -82,3 +82,19 @@ The palette data is sent by VRAM-Transfer (4 KBytes).
 
 Each Palette consists of four 16-bit color definitions (8 bytes). Note:
 The data is stored at 3000h-3FFFh in SNES memory.
+
+## SGB Command $19 — PAL_PRI
+
+If the player overrides the active palette set (a pre-defined or the custom one), it stays in effect until the smiley face is selected again, or the player presses the X button on their SNES controller.
```

Comment:
_A pre-defined or THE custom one_ sounds a bit unclear here... Maybe "the pre-defined or the the custom one"? or "a pre-defined or a custom one"?

Reply:
There are several pre-defined palettes, but only a single custom one.

Reply:
And those are the ones getting "overridden" by the player?

Reply:
This is the single custom one:
![Screenshot_20230118_211531](https://user-images.githubusercontent.com/122655324/213296585-7a349bab-1765-4bd2-9123-da854e1525b0.png)

These are the predefined ones (4 pages of 8):
![Screenshot_20230118_211604](https://user-images.githubusercontent.com/122655324/213296589-0388373d-9485-45d1-a015-7b295b643e56.png)

These are user-selectable palettes, which can be overridden when PAL_PRI's flag is set

Reply:
Okay, I was just trying to understand what information we need to convey by specifying that the active one can be either the custom or one of the default ones, which I guess is just the standard behaviour, hence it doesn't add any information to the "active palette set" so it could be removed?

Reply:
Would it be worth describing system palettes and user palettes above this? Then the initial part of the sentence would be

If the player overrides the system palettes (with a pre-defined or the custom user palette),

Reply:
Yes, but I don't want to hold PR for this, so I'll merge this in the the current status and we can take care of improving this later.